crates/bevy_ecs/src/schedule/condition.rs

```diff
@@ -105,4 +108,37 @@ pub mod common_conditions {
             None => false,
         }
     }
+
+    /// Generates a  [`Condition`](super::Condition) that inverses the result of passed one.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use bevy_ecs::prelude::*;
+    /// // Building a new schedule/app...
+    /// let mut sched = Schedule::default();
+    /// sched.add_system(
+    ///         // This system will always run.
+    ///         my_system.run_if(not(always_false))
+    ///     )
+    ///     // ...
+    /// #   ;
+    /// # let mut world = World::new();
+    /// # sched.run(&mut world);
+    ///
+    /// // A condition that always returns false.
+    /// fn always_false() -> bool {
+    ///    false
+    /// }
+    ///
+    /// # fn my_system() {}
```

Comment:
```suggestion
    ///         // This system will never run.
    ///         my_system.run_if(not(always_true))
    ///     )
    ///     // ...
    /// #   ;
    /// # let mut world = World::new();
    /// # sched.run(&mut world);
    ///
    /// // A condition that always returns true.
    /// fn always_true() -> bool {
    ///    true
    /// }
    /// #
    /// # fn my_system() { unreachable!() }
```

This should make the doctest actually function as a test.